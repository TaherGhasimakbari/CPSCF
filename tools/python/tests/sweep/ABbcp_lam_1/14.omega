 format  1  0
dim                 
                   1
crystal_system      
          'lamellar'
N_cell_param        
                   1
cell_param          
    1.4568255865E+00
group_name          
                '-1'
N_monomer           
                   2
N_star              
                  17
  9.600000003687E+00  5.400000007313E+00       0     1
 -5.165487187352E+00  2.952065948897E+00       1     2
 -5.156745056133E-01  1.329871610470E+00       2     2
  6.083688706360E-01  5.163083041403E-01       3     2
  1.765942984992E-01 -2.076606889422E-02       4     2
 -5.026757483352E-02 -9.910110460064E-02       5     2
 -4.354623037441E-02 -3.749312026926E-02       6     2
 -6.601279746998E-03  3.058453335488E-04       7     2
  3.908159051852E-03  5.329521882579E-03       8     2
  2.116012223817E-03  1.814808487492E-03       9     2
  1.645511768678E-04 -7.461766297388E-05      10     2
 -2.116896854258E-04 -2.513308909819E-04      11     2
 -8.547273938383E-05 -7.195759646360E-05      12     2
 -1.612802604635E-06  6.703821924805E-06      13     2
  9.107205296217E-06  1.018589692746E-05      14     2
  2.654542080128E-06  2.044675520227E-06      15     2
 -1.510345967923E-07 -5.666486318035E-07      16     1
