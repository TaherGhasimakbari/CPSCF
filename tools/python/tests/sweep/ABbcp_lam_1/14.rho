 format  1  0
dim                 
                   1
crystal_system      
          'lamellar'
N_cell_param        
                   1
cell_param          
    1.4568255865E+00
group_name          
                '-1'
N_monomer           
                   2
N_star              
                  17
  3.600000002235E-01  6.399999997765E-01       0     1
  2.705851043456E-01 -2.705851043461E-01       1     2
  6.151820382701E-02 -6.151820382416E-02       2     2
 -3.068685544797E-03  3.068685549737E-03       3     2
 -6.578678906207E-03  6.578678909307E-03       4     2
 -1.627784323688E-03  1.627784324406E-03       5     2
  2.017703360619E-04 -2.017703366473E-04       6     2
  2.302375018357E-04 -2.302375023095E-04       7     2
  4.737876075971E-05 -4.737876084941E-05       8     2
 -1.004012445865E-05  1.004012450171E-05       9     2
 -7.972294492702E-06  7.972294524647E-06      10     2
 -1.321373416858E-06  1.321373430897E-06      11     2
  4.505047532793E-07 -4.505047418147E-07      12     2
  2.772207364277E-07 -2.772207357184E-07      13     2
  3.595632675426E-08 -3.595633675278E-08      14     2
 -2.032886178196E-08  2.032885683180E-08      15     2
 -1.385375104912E-08  1.385375111851E-08      16     1
