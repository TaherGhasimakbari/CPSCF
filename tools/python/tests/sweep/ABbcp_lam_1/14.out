 format  1  0

MONOMERS            
N_monomer           
                   2
kuhn                
    1.0000000000E+00    1.0000000000E+00

CHAINS              
N_chain             
                   1
N_block             
                   2
block_monomer       
                   1                   2
block_length        
    3.6000000022E-01    6.3999999978E-01

COMPOSITION         
ensemble            
                   0
phi_chain           
    1.0000000000E+00

INTERACTION         
interaction_type    
               'chi'
chi                 
    1.5000000011E+01

UNIT_CELL           
dim                 
                   1
crystal_system      
          'lamellar'
N_cell_param        
                   1
cell_param          
    1.4568255865E+00

DISCRETIZATION      
ngrid               
                  32
chain_step          
    1.0000000000E-02

FILE_PREFIXES       
input_prefix        
               'in.'
output_prefix       
              'ref/'

BASIS               
group_name          
                '-1'

ITERATE             
max_itr             
                  20
error_max           
    1.0000000000E-08
domain              
                   T
itr_algo            
                'NR'
N_cut               
                  17

SWEEP               
s_max               
    2.0000000000E+01
d_block_length      
   -1.0000000000E-02    1.0000000000E-02
end_increments      

FINISH              

THERMO              
f_Helmholtz         
    2.3351243592E+00
f_homo              
    2.4560000035E+00
pressure            
    3.3001565563E+00
mu_chain            
    5.6352809155E+00
stress              
    3.0297522098E-11

STATISTICS          
N_star              
                  17
Final Error         
    3.0297522098E-09
Iterations          
                   8
Basis Time          
    2.6167106628E-01
SCF Time            
    6.1519145966E-02
